pkg/epp/handlers/request.go

```diff
@@ -28,6 +28,12 @@ import (
 	errutil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/error"
 )
 
+const (
+	DestinationEndpointHintMetadataNamespace = "envoy.lb"
+	DestinationEndpointHintKey               = "x-gateway-destination-endpoint"
+	FairnessIDHeaderKey                      = "x-gateway-inference-fairness-id"
```

Comment:
Consider `FlowFairnessID` instead of `FairnessIDHeaderKey` if it's defined in a new `metadata` pkg.

We should also consider having consistency across the metadata consts, i.e., dropping the `key` suffix and using godocs to explain it's a header or metadata key.